<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureGraphics</class>
 <widget class="QWidget" name="ConfigureGraphics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Graphics</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="toggle_hw_renderer">
          <property name="text">
           <string>Enable hardware renderer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="toggle_shader_jit">
          <property name="text">
           <string>Enable shader JIT</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="toggle_scaled_resolution">
          <property name="text">
           <string>Enable scaled resolution</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="toggle_vsync">
          <property name="text">
           <string>Enable V-Sync</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox2">
     <property name="title">
      <string>Layout</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label1">
            <property name="text">
             <string>Screen Layout:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="layout_combobox">
            <item>
             <property name="text">
              <string notr="true">Default</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">Single Screen</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">Large Screen</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="swap_screen">
          <property name="text">
           <string>Swap Screens</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
